mlrun/datastore/targets.py

```diff
@@ -1644,15 +1660,20 @@ def as_df(
         db_path, table_name, _, _, _, _ = self._parse_url()
         engine = sqlalchemy.create_engine(db_path)
         with engine.connect() as conn:
-            df = pd.read_sql(
-                f"SELECT * FROM {self.attributes.get('table_name')}",
-                con=conn,
-                parse_dates=self.attributes.get("time_fields"),
+            # TODO : filter as part of the query
+            df = filter_df_start_end_time(
+                pd.read_sql(
+                    f"SELECT * FROM {self.attributes.get('table_name')}",
+                    con=conn,
+                    parse_dates=self.attributes.get("time_fields"),
```

Comment:
Preexisting, but I'm pretty sure that `"time_fields"` (plural) is a bug, and it should be `"time_field"`.

Reply:
opened a ticket [ML-3929](https://jira.iguazeng.com/browse/ML-3929)